index.html

```diff
@@ -0,0 +1,216 @@
+        <ul class="container content menu">
+            <li class="textPlaceholder menu-item"><a href="#projects" aria-label="Portfolio"> Portfolio</a></li>
+            <li class="textPlaceholder menu-item"><a href="#about" aria-label="About"> About</a></li>
+            <li class="textPlaceholder menu-item"><a href="#contact" aria-label="Contact"> Contact</a></li>
+        </ul>
+    </section>
+    <section id="presentation" class="container">
+        <div class="container content">
+            <div>
+                <h1>
+                    Hey There. <br> I’m Stephen <span class="dot"></span>
+                </h1>
+                <h2>
+                    I’m a Software Developer 
+                </h2>
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+            </div>
+            <ul class="container socialGroup">
+                <li class="social-media">
+                    <a rel="noopener noreferrer" id="githubButton" href="https://github.com/ssenkooto1" target="_blank"
+                        aria-label="GitHub"><img src="icons/gitHub.svg" alt="Github.com Icon"></a>
+                </li>
+                <li class="social-media">
+                    <a rel="noopener noreferrer" id="linkedinButton"
+                        href="www.linkedin.com/in/stephen-ssenkooto-00a036b1" target="_blank"
+                        aria-label="LinkedIn"><img src="icons/linkedin.svg" alt="LinkedIn.com Icon"></a>
+                </li>
+                <li class="social-media">
+                    <a rel="noopener noreferrer" id="angelistButton" href="#" aria-label="Angelist"><img
+                            src="icons/angelist.svg" alt="Angelist.com icon"></a>
+                </li>
+                <li class="social-media">
+                    <a rel="noopener noreferrer" id="twitterButton" href="https://twitter.com/SsenkootoSteph1"
+                        target="_blank" aria-label="Twitter"><img src="icons/twitter.svg" alt="Twitter.com icon"></a>
+                </li>
+                <li class="social-media">
+                    <a rel="noopener noreferrer" id="mediumButton" href="#" aria-label="Medium"><img
+                            src="icons/medium.svg" alt="Medium.com Icon"></a>
+                </li>
+            </ul>
+        </div>
+    </section>
+    <section id="projects" class="container works">
+        <div class="container works">
+            <h2> My Recent Works </h2>
+            <span class="line"></span>
+        </div>
+        <div class="gridContainer">
+        </div>
+    </section>
+    <section id="about" class="container">
+        <div class="container content about">
+            <h1> About me </h1>
```

Comment:
- [ ] Kindly make sure you don't have more than 1 `h1` tag in your code as it confuses the [SEO](https://developer.mozilla.org/en-US/docs/Glossary/SEO) (Search Engine Optimization) when searching for your page🙄. I'd suggest you use other `hx` tag `h2-h6`👍